pyhub/mcptools/google/sheets/client.py

```
@@ -0,0 +1,244 @@
+logger = logging.getLogger(__name__)
+
+
+def handle_api_errors(func):
+    """Google API 에러 처리 데코레이터"""
+
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except RefreshError:
+            raise AuthenticationError("인증 토큰을 갱신할 수 없습니다. 다시 로그인하세요.")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)